src/pages/library/dataBrowser-utils.ts

```diff
@@ -40,34 +57,37 @@ export const getDatasetReleasePoliciesDisplayInformation = dataUsePermission =>
   )
 }
 
-export const DatasetReleasePolicyDisplayInformation = ({ dataUsePermission }) => {
+export const DatasetReleasePolicyDisplayInformation = ({ 'TerraDCAT_ap:hasDataUsePermission': dataUsePermission }: { 'TerraDCAT_ap:hasDataUsePermission': string }) => {
   const { label, description } = getDatasetReleasePoliciesDisplayInformation(dataUsePermission)
   return h(Fragment, [
     label,
     description && div({ style: { fontSize: '0.625rem', lineHeight: '0.625rem' } }, [description])
   ])
 }
 
-export const isExternal = dataset => Utils.cond(
+export const isExternal = (dataset: DatasetResponse) => Utils.cond(
   [isWorkspace(dataset), () => false],
   [isDatarepoSnapshot(dataset), () => false],
-  () => true)
+  () => true) as boolean
```

Comment:
I wonder if it's possible to write (or find) a typesafe version of `cond()`. Or maybe there's a more idiomatic way to write this in typescript?

Reply:
It is probably possible, but agreement is within the Terra-UI group to not type shared code until we are ready for the full rollout.

Reply:
There's no need to change the original `cond` if we want to add a ts one as well.